src/SumoCoders/FrameworkCoreBundle/Composer/ScriptHandler.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Composer;
+
+use Composer\Script\Event;
+
+class ScriptHandler
+{
+    /**
+     * Install NPM
+     *
+     * @param Event $event
+     */
+    public static function installNPM(Event $event)
+    {
+        static::runCommandOnlyInDevMode('npm install', $event);
+    }
+
+    /**
+     * Install Bower dependencies
+     *
+     * @param Event $event
+     */
+    public static function installBower(Event $event)
+    {
+        static::runCommandOnlyInDevMode('bower install', $event);
+    }
+
+    /**
+     * Run grunt build
+     *
+     * @param Event $event
+     */
+    public static function gruntBuild(Event $event)
+    {
+        static::runCommandOnlyInDevMode('grunt build', $event);
+    }
+
+    /**
+     * Running the given command only when we are in dev-mode
+     * The output will be send directly to the output buffer
+     *
+     * @param string $command
+     * @param Event  $event
+     */
+    public static function runCommandOnlyInDevMode($command, Event $event)
```

Comment:
Zoals in de vorige comment beschreven, ik zou hier enkel IO injecten (en natuurlijk een flag dev/of niet), niet het volledige Event object. Hierdoor krijg je betere separation of concerns.